Resources/iperf-Bridging-Header.h

```diff
@@ -0,0 +1,7 @@
+//
+//  Use this file to import your target's public headers that you would like to expose to Swift.
+//
+
+#import "IPFTestRunner.h"
+#import "IPFTestRunnerConfiguration.h"
+#import "IPFHelpViewController.h"
```

Comment:
If we're going Swift for the UI we'd probably want to move the help view controller as well, though maybe not right now

Reply:
Can switch it. It looks like a small view controller

Reply:
Switched to swift

Reply:
Switched to Swift